client/components/checkbox/README.md

```diff
@@ -0,0 +1,32 @@
+Checkbox
+========
+
+Checkbox is a React component used to render a checkbox input field. It is essentially an enhanced version of `<input type="checkbox" />`, using Gridicon as check mark.
+
+## Usage
+
+Refer to the following code snippet for a typical usage example:
+
+```jsx
+<Checkbox
+	checked={ this.state.showSubtitles }
+	onChange={ this._toggleShowSubtitles }>
+	Show subtitles
+</Checkbox>
+```
+
+## Props
+
+Props not listed below will be passed automatically to the rendered range input element.
```

Comment:
This is probably something that's widely-enough known and universal to React components that we probably don't need it here either. Leaving it in could confuse people who expect this behavior, thinking that the mere mention of it might indicate that something is _not_ normal about how the props get passed.